src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -317,6 +317,10 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
 
         // need to load the pending item
         this.fetchFeedItems();
+
+        if (onTaskAssignmentUpdate) {
+            onTaskAssignmentUpdate(taskId, taskAssignmentId, status);
```

Comment:
It looks like we all onTaskUpdate in the success callback of the updateTask method. Can we stay consistent with that here, as well?

Reply:
Updated, thanks for the feedback!